Source/JavaScriptCore/wasm/WasmOperationsInlines.h

```diff
@@ -54,9 +54,11 @@ inline EncodedJSValue arrayNew(Instance* instance, uint32_t typeIndex, uint32_t
     const Wasm::TypeDefinition& arraySignature = instance->module().moduleInformation().typeSignatures[typeIndex]->expand();
     ASSERT(arraySignature.is<ArrayType>());
     Wasm::FieldType fieldType = arraySignature.as<ArrayType>()->elementType();
+    auto arrayRTT = instance->module().moduleInformation().rtts[typeIndex];
```

Comment:
The RTT now has to be supplied on object construction right here, rather than when the structure is created. This isn't a problem because all the RTTs are tracked in the module information.